src/openloco/graphics/gfx.h

```diff
@@ -4,6 +4,8 @@
 #include "../localisation/stringmgr.h"
 #include "../openloco.h"
 
+#define LOCO_G1_ELEMENT_COUNT 0x1A10
```

Comment:
Does this hold for the other variants (Steam/GOG/…) as well?

Reply:
GOG, yes. Steam... no, it's `0x380F`. I'll need to investigate that.

Reply:
Are you sure it's not 0x101A and 0xF38? Anyhow, I'd say log a warning in case g1 count doesn't match expected count. Probably best to still use the other one for array size, in case something depends on it.